script/Ciel/Day 6/06_01_CIEL06_7.txt

```diff
@@ -330,17 +331,17 @@ Everyone can go pat themselves on the back for resolving the incident in 'this c
 -- 「帰りたいのは山々だがな。片足つっこんだ手前、最後まで面倒見るしかねぇだろ。
 // 片足を突っ込む -> to get a little involved
 "Oh, I'd love nothin' more than to get outta here.
- Unfortunately we already got our hands dirty, so now we gotta see it through.
+ Unfortunately, we already got our hands dirty, so now we gotta see it through.
 }
 [sha:e3e115e68e110af08b59e82305a9b29f57aabd7a]{
 -- Page 274, Offset 13880.
 -- ……けどなあ、お姫さま。アンタはいいのかよ、それで」
-Still, princess... You sure you're doing okay?"
+Still, Princess... You sure you're doing okay?"
 }
 [sha:7a0bd8125aa6e2ce19f54628eddd9d95d4101763]{
 -- Page 275, Offset 13881.
 -- 「へえ―――いいって、何が？」
-"Oh? Okay how?"
+"Oh? Okay, how?"
```

Comment:
I'd revert - "Okay how?" to me is "what is it that makes you ask I'm okay?" whereas "okay, how" is more "okay I acknowledge a thing you've just said, how [asking new question]?"